superset3/assets/superset_config.py

```diff
@@ -221,3 +223,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 
 if os.environ.get("OVERRIDE_APP_ICON"):
     APP_ICON = "/static/assets/images/logo.png"
+
+if os.environ.get("CUSTOM_COLOR_PALETTE"):

```

Comment:
Can we add this variable to the env.example file

Reply:
good idea, added